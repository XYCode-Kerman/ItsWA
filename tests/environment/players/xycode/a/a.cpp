#include <bits/stdc++.h>
#define ll long long
#define int long
using namespace std;
const ll N = 10;

signed main() {
    freopen("a.in","r",stdin);
    freopen("a.out","w",stdout);
    
    string s;cin>>s;
    cout<<s<<' '<<s;
    return 0;
}