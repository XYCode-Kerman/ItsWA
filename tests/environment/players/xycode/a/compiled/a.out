114514 114514